docs/cmd/kn_domain_create.md

```diff
@@ -10,8 +10,8 @@ kn domain create NAME
 
 ```
 
-  # Create a domain mappings 'hello.example.com' for Knative service 'hello'
-  kn domain create hello.example.com --ref hello
+  # Create a domain mappings 'hello.example.com' for Knative service 'hello' with TLS secret set
+  kn domain create hello.example.com --ref hello --tls my-tls-secret
```

Comment:
could we call `my-tls-secret` --> `my-cert-secret` ? (as TLS is more the overall mechanism that is enabled by this option, but the secret holds the server certificate used during TLS communication, so we can be more specific here)